delete-wav.js

```diff
@@ -0,0 +1,9 @@
+module.exports = function deleteWav(pathToFile) {
+  var fs = require('fs')
+  fs.unlink(pathToFile, (err) => {
```

Comment:
Since you are using `unlink` in a synchronous way, change this to `fs.unlinkSync`